content/docs/introduction/glossary.md

```diff
@@ -52,8 +52,8 @@ to email, Pagerduty, Slack etc.
 
 ### Promdash
 
-[Promdash](../../visualization/promdash/) is a Ruby-on-Rails dashboard builder for Prometheus.
-It is similar at a high-level to Grafana, but is specific to Prometheus.
+Promdash is a deprecated Ruby-on-Rails dashboard builder for Prometheus. It was
```

Comment:
That's not what we have in master. I think we should merge master into next-release first.